CHANGELOG_PENDING.md

```diff
@@ -1,4 +1,7 @@
 ### SDK Features
+* `services/transcribestreamingservice`: Support for Amazon Transcribe Streaming ([#3048](https://github.com/aws/aws-sdk-go/pull/3048))
+  * The SDK now supports the Amazon Transcribe Streaming APIs by utilizing event stream encoding over HTTP/2
+  * See [Amazon Transcribe Developer Guide](https://docs.aws.amazon.com/transcribe/latest/dg)
```

Comment:
Looks good, should include fixes https://github.com/aws/aws-sdk-go/issues/2487 as well.